documentation/serving_doc.md

```diff
@@ -0,0 +1,29 @@
+# Serving generated doc
```

Comment:
@Danielwsx64 WDYT of adding a new markdown file, add this as a section of the moduledoc of xcribe. In that way it will be available at the hexdocs in a easy way.

Reply:
I just saw that you include it in the final doc as a extra section. I think that is already enough.

However I personally prefer to have it in the main moduledoc. Seems to be a more appropriate place.

Reply:
I created a separate section because I think serve generated doc with Xcribe is a bonus not a core feature (usually the documentation is placed in a commercial site). Any way there's a link in the bottom of Xcribe moduledoc.